### ISSUE TITLE
rclify parameters failed

### ISSUE THREAD

**Issue by USeebi** [suurjaak/rosros#1]
I tried rosros package to make an existing ROS2 package from 
https://github.com/joshnewans/ball_tracker/blob/main/ball_tracker/follow_ball.py to be useable under ROS1. However I got following errors,

```
Traceback (most recent call last):
  File "/home/orangepi/action_ws/src/opencv/src/detect_ball_3d.py", line 96, in <module>
    detect_ball_3d = DetectBall3d()
  File "/home/orangepi/action_ws/src/opencv/src/detect_ball_3d.py", line 28, in __init__
    super().__init__('detect_ball_3d')
  File "/home/orangepi/Downloads/rosros/src/rosros/rclify/node.py", line 117, in __init__
    self._parameter_overrides = {k: Parameter(k, value=v)
  File "/home/orangepi/Downloads/rosros/src/rosros/rclify/node.py", line 118, in <dictcomp>
    for k, v in ros1.get_params(nested=False)}
ValueError: too many values to unpack (expected 2)
```

What did I miss? The diff between the original source and the ROS1/ROS2 script are follows:

```
0a1,3
> #!/usr/bin/env python
>
> # https://github.com/joshnewans/ball_tracker/
15,16c18,20
< import rclpy
< from rclpy.node import Node
---
> #import rclpy
> from rosros import rclify as rclpy
> #from rclpy.node import Node
21c25
< class DetectBall3d(Node):
---
> class DetectBall3d(rclpy.node.Node):
28c32
<         self.ball2d_sub  = self.create_subscription(Point,"/detected_ball",self.ball_rcv_callback, 10)
---
>         self.ball2d_sub  = self.create_subscription(Point,"/blob/point_blob",self.ball_rcv_callback, 10)
35c39
<         self.declare_parameter("camera_frame",'camera_link_optical')
---
>         self.declare_parameter("camera_frame",'usb_cam')
83a88
>         self.get_logger().info('Publishing 3d position: "%s"' % m.pose.position)
88,93c93,99
< def main(args=None):
<     rclpy.init(args=args)
<     detect_ball_3d = DetectBall3d()
<     rclpy.spin(detect_ball_3d)
<     detect_ball_3d.destroy_node()
<     rclpy.shutdown()
---
> #def main(args=None):
>     #rclpy.init(args=args)
> rclpy.init()
> detect_ball_3d = DetectBall3d()
> rclpy.spin(detect_ball_3d)
>     #detect_ball_3d.destroy_node()
>     #rclpy.shutdown()
```

**Comment by suurjaak** [comment]
Confirmed, there is a bug in rclify. Your code is correct. 

I will release a fix presently.

**Comment by suurjaak** [comment]
Fixed in the newly released v0.2.6.

Thank you for reporting this.
